### ISSUE TITLE
New SoapySDR commit breaks RTL_443

### ISSUE THREAD

**Issue by tranzmatt** [merbanan/rtl_433#1278]
The latest [commit ](https://github.com/pothosware/SoapySDR/commit/f722f9ce5b629c3c44401a9bf628b3f8e67a9695#diff-0086a1586b78d24d3bcdb0c37ded0163)of SoapySDR breaks RTL 443 if compiled against it.  The culprit is the removal of a compatibility macro in Device.h that's used by sdr.c

```
/pybombs/src/rtl_433/src/sdr.c:692:9: error: too many arguments to function ‘SoapySDRDevice_setupStream’
          if (SoapySDRDevice_setupStream(dev->soapy_dev, &dev->soapy_stream, SOAPY_SDR_RX, format, NULL, 0, &stream_args) != 0) {
              ^~~~~~~~~~~~~~~~~~~~~~~~~~
     In file included from /pybombs/src/rtl_433/src/sdr.c:28:0:
     /pybombs/include/SoapySDR/Device.h:295:31: note: declared here
      SOAPY_SDR_API SoapySDRStream *SoapySDRDevice_setupStream(SoapySDRDevice *device,
                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~

```

**Comment by zuckschwerdt** [comment]
Yes, we currently need SoapySDR 0.7. I guess we can only update our code once 0.8 is released.
Maybe we could detect the (missing) macro and add a shim.

**Comment by zuckschwerdt** [comment]
Should be fixed with 056804c now.
